test/schema/errorResponse.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id"                  : "errorResponse.json",
+  "$schema"             : "http://json-schema.org/draft-04/schema#",
+  "title"               : "errorResponse-2.0",
+  "type"                : "object",
+  "properties"          : {
+    "jsonrpc"                   : {"type": "string"},
+    "id"                        : {"type": "number"},
```

Comment:
id is a string not a number.

Reply:
the auto-generated id is number, but if needed i can change it to string
example if requests: [{'jsonrpc': '2.0', 'method': 'POST', 'params': {}, 'id': 88}]

Reply:
I mean that the JSON schema says: {"type": "number"}, but it should be {"type": "string"}

Because that is how the spec is.